### PR TITLE
sqlite3: add German translation

### PR DESCRIPTION
**Pull request by LukBukkit**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** Not known

### CODE REVIEW

File: pages.de/common/sqlite3.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+> Das Kommandozeileninterface für SQLite 3, welches eine eigenständige dateibasierte eingebettete SQL-Engine ist.
+> Weitere Informationen: <https://sqlite.org>.
+
+- Starte eine interaktive Shell mit einer neuen Datenbank:
+
+`sqlite3`
+
+- Öffne eine interaktive Shell mit einer existierenden Datenbank:
+
+`sqlite3 {{pfad/zur/datenbank.sqlite3}}`
```
Comment (kbdharun) [inline pages.de/common/sqlite3.md:12]:
```suggestion
`sqlite3 {{pfad/zu/datenbank.sqlite3}}`
```

File: pages.de/common/sqlite3.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Führe ein SQL Statement auf einer existierenden Datenbank aus und beende die Ausführung danach:
+
+`sqlite3 {{pfad/zur/datenbank.sqlite3}} '{{SELECT * FROM einer_tabelle;}}'`
```
Comment (kbdharun) [inline pages.de/common/sqlite3.md:16]:
```suggestion
`sqlite3 {{pfad/zu/datenbank.sqlite3}} '{{SELECT * FROM einer_tabelle;}}'`
```

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution. The translation LGTM, I have some suggestions for this page.

**Comment by LukBukkit** [comment]
Good catch, I didn't notice that most of the pages simply use 'zu' instead of 'zur' 👍

**Review by kbdharun — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**